packages/inter-protocol/test/vaultFactory/test-vaultLiquidation.js

```diff
@@ -159,9 +162,10 @@ const setupServices = async (
     quoteInterval,
     unitAmountIn,
     {
-      StartFreuency: startFrequency,
```

Comment:
fwiw, if the function didn't [destructure a default value](https://github.com/Agoric/agoric-sdk/blob/d7a6de875ac7df2acb599f2081dbdcd325b9e596/packages/inter-protocol/test/vaultFactory/vaultFactoryUtils.js#L72), TS would have detected this typo